backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -712,4 +718,57 @@ export class ContributionResolver {
       }
     })
   }
+
+  @Authorized([RIGHTS.REJECT_CONTRIBUTION])
+  @Mutation(() => Boolean)
+  async rejectContribution(
+    @Arg('id', () => Int) id: number,
+    @Ctx() context: Context,
+  ): Promise<boolean> {
+    const contributionToUpdate = await DbContribution.findOne({ id })
+    // TODO: Check
```

Comment:
`confirmedAt`  must be null
`deniedAt` must be nulll

Reply:
Thanks! Good catch!